runtime/server/column_api_test.go

```diff
@@ -0,0 +1,61 @@
+package server
+
+import (
+	"context"
+	"github.com/rilldata/rill/runtime/api"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestServer_GetTopK(t *testing.T) {
+	server, instanceId, err := GetTestServer()
+
+	server.QueryDirect(context.Background(), &api.QueryDirectRequest{
```

Comment:
You should do `_, err = server.QueryDirect...` to get the error and check it

Reply:
done